### ISSUE TITLE
Project maintenance

### ISSUE THREAD

**Issue by fabiosantoscode** [zekesonxx/pinecone#1]
Greetings!

I love this project, and want to integrate it into [ecmacomp](https://github.com/fabiosantoscode/ecmacomp/) so that you can require lua files from other languages.

Do you think it's up to the task?

Also, do you accept PRs for this project? In case some bug pops up we'll try and fix it.

**Comment by zekesonxx** [comment]
I think you misunderstand the scope of this project. pinecone compiles code JS->Lua, not the other way around.

Pinecone is also not a drop-in JS->Lua converter; you have to write your JS code with the intention of it compiling to Lua.

I'm all for seeing more work done on pinecone, and would love to see others get use out of it, but I don't see how it relates in any way to ecmacomp.

**Comment by fabiosantoscode** [comment]
Right, I think I misunderstood it. We don't need to compile all languages to all other languages, instead we're only compiling to JS.
